gradle/scripts/lib/java-alpn.gradle

```diff
@@ -14,7 +14,7 @@ configure(projectsWithFlags('java')) {
         }
 
         dependencies {
-            alpnAgent 'org.mortbay.jetty.alpn:jetty-alpn-agent'
+            alpnAgent "org.mortbay.jetty.alpn:jetty-alpn-agent:${managedVersions['org.mortbay.jetty.alpn:jetty-alpn-agent']}"
```

Comment:
There are two problems with `libs...`. 
- We don't assume that the version of Jetty ALPN is defined at `libs.versions.jetty.alpn.agent`.
- It could be a breaking change for the users who do not use `dependencies.toml`.